### PR TITLE
games-strategy/freeorion: stable release 0.4.10.2

### PR DESCRIPTION
**Pull request by cengique**
Tested-by: Cengiz Gunay <cengique@users.sf.net>
Package-Manager: Portage-3.0.30, Repoman-3.0.3
Signed-off-by: Cengiz Gunay <cengique@users.sf.net>
Please check @NP-Hardass

### CODE REVIEW

**Comment by cengique** [comment]
I don't think there are any issues with this other than a Python upgrade check. Can we move it along @NP-Hardass @DarthGandalf @thesamesam ?

File: games-strategy/freeorion/freeorion-0.4.10.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,77 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```
Comment (DarthGandalf) [inline games-strategy/freeorion/freeorion-0.4.10.2.ebuild:4]: Why not 8?

File: games-strategy/freeorion/freeorion-0.4.10.2.ebuild [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+src_prepare() {
+	sed -e "s/-O3//" -i CMakeLists.txt || die
+
+	cmake_src_prepare
+}
+
+src_configure() {
+	local mycmakeargs=(
+		-DCMAKE_BUILD_TYPE=Release
```
Comment (DarthGandalf) [inline games-strategy/freeorion/freeorion-0.4.10.2.ebuild:65]: "Release" is the default with EAPI 8

**Review by DarthGandalf — COMMENTED** [review]
I didn't try to build it, but seems fine, other than EAPI?

Why do you need my opinion anyway? :) I'm not maintaining this package, and didn't join games project yet either

File: games-strategy/freeorion/freeorion-0.4.10.2.ebuild (hunk 2 above)
Comment (DarthGandalf) [reply games-strategy/freeorion/freeorion-0.4.10.2.ebuild:65]: Erm, RelWithDebInfo.

But then, the question is why you don't want RelWithDebInfo?

**Pull request closed without merging**